lib/cog/relay/tracker.ex

```diff
@@ -81,7 +119,20 @@ defmodule Cog.Relay.Tracker do
   def relays(tracker, bundle_name) do
     tracker.map
     |> Map.get(bundle_name, MapSet.new)
+    |> MapSet.difference(tracker.disabled)
     |> MapSet.to_list
   end
 
+  defp in_tracker?(tracker, relay_id) do
+    relays = Enum.reduce(tracker.map, MapSet.new(), fn({_, relays}, acc) ->
```

Comment:
Might be able to simplify this a bit and make the whole thing a pipeline by calling `Map.values(tracker.map)` first.